src/translations/en.json

```
@@ -905,6 +905,8 @@
           "entity_id": "Entity ID",
           "unit_of_measurement": "Unit of Measurement",
           "precipitation_unit": "Precipitation unit",
+          "precision": "Precision",
+          "precision_default": "Use default from integration",
```

```suggestion
          "precision_default": "Use default",
```
Just "use default" ?